mantidproject/mantid
# Add algorithm for tomographic reconstruction method(s)

This adds an algorithm that implements one tomographic reconstruction method. The method is FBP, in the implementation from tomopy (see issue #15971).

**To test:**
- Code review
- Check that all the ingredients of the algorithm look good, including:
  - documentation
  - unit, doc tests

The algorithm and code is organized so that more reconstruction methods could be added in the future.
The code that comes originally and is still very-ish close to the original code of tomopy (files utils.c and fbp.c) has been added as C and not C++ (see changes in the CMakeLists.txt). I'd prefer this to ease up/down-stream code updates with tomopy.

Fixes #15971.

Release notes: https://github.com/mantidproject/mantid/pull/16027/commits/4ffd8348182b8b4f2e024578cbfaa3f0234b9a86

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
